packages/cli/src/oclif/commands/scaffold.js

```diff
@@ -0,0 +1,217 @@
+const _ = require('lodash');
+const path = require('path');
+const colors = require('colors/safe');
+
+const { flags } = require('@oclif/command');
+
+const BaseCommand = require('../ZapierBaseCommand');
+const { buildFlags } = require('../buildFlags');
+const utils = require('../../utils');
+
+class ScaffoldCommand extends BaseCommand {
+  writeTemplateFile(templatePath, templateContext, dest) {
+    const destPath = path.join(process.cwd(), `${dest}.js`);
+    const preventOverwrite = !this.flags.force;
+
+    if (preventOverwrite && utils.fileExistsSync(destPath)) {
+      const [location, filename] = dest.concat('.js').split('/');
+      return Promise.reject(
+        [
+          `File ${colors.bold(filename)} already exists within ${colors.bold(
+            location
+          )}.`,
+          'You could:',
+          '1. Choose a different filename',
```

Comment:
can you indent these so they look nice on the CLI? maybe add an extra line break as well. I'm picturing: 

```
You could: 

  1. asdf
  2. asdf

$
```